Userland/Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -695,35 +730,17 @@ static ThrowCompletionOr<Value> pad_string(VM& vm, Utf16String string, Value max
     if (int_max_length <= string_length)
         return PrimitiveString::create(vm, move(string));
 
-    // 5. If fillString is undefined, let filler be the String value consisting solely of the code unit 0x0020 (SPACE).
+    // 5. If fillString is undefined, set fillString to the String value consisting solely of the code unit 0x0020 (SPACE).
+    // FIXME: Set fillString to the String value.
```

Comment:
Is there a way to set the `JS::Value` "fillString" here to the String value? Can't seem to find any examples in the codebase